lib/Backends/Habana/Habana.cpp

```diff
@@ -564,26 +541,26 @@ static std::unique_ptr<synConvolutionParams> makeSynConvolutionParams(
   return params;
 }
 
-static std::unique_ptr<synPoolParams>
+static std::unique_ptr<ns_SpatialReduction::Params>
```

Comment:
we actually have dilation param now, but we can fix it later.

Reply:
I thought we only have dilation for convolution?